keymaps/ava.json

```diff
@@ -1,6 +1,7 @@
 {
 	"atom-workspace": {
-		"ctrl-alt-r": "ava:run",
-		"ctrl-alt-a": "ava:toggle"
+		"ctrl-alt-shift-a": "ava:run",
+		"ctrl-alt-a": "ava:run-all",
+		"ctrl-alt-t": "ava:toggle"
```

Comment:
I think we should drop this shortcut. It's already covered by running. We can rather add support for pressing Esc when the pane is active to close it.